code/upaya/app/services/id_proofer.rb

```diff
@@ -1,102 +0,0 @@
-require 'exceptions'
```

Comment:
Yes it is. For some reason, before the switch, the app was not complaining about the missing `Exceptions` file. But now it is, so I removed it. I could have added the Exceptions file to fix the issue, but I prefered to remove IdProofer because the Id Proofing feature isn't part of the app yet.